### ISSUE TITLE
When ripping an audio CD, the stage fails to load properly. (Docker)

### ISSUE THREAD

**Issue by Anti-Apple4life** [automatic-ripping-machine/automatic-ripping-machine#837]
When ripping an audio CD, the stage fails to load properly, resulting in the following error:

![Not expecting `NoneType`](https://i.imgur.com/eSjvukL.png)

ARM still rips the CD, but you have to see the progress through the log.

_Originally posted by @haxys in https://github.com/automatic-ripping-machine/automatic-ripping-machine/issues/522#issuecomment-1312628798_

**Comment by microtechno9000** [comment]
agree this needs fixing as its hard to see what progress is being made. will add it to the list of bugs to fix.

**Comment by Anti-Apple4life** [comment]
UPDATE: 
This is caused by (apparently) incorrect disk mounting, as on the few times it does not happen, the log shows the disk mounted successfully.

```
[05-13-2023 01:10:26] INFO ARM: Mounting disc to: /mnt/dev/sr1
[05-13-2023 01:10:26] ERROR ARM: Mounting failed! Rip might have problems
```

**Comment by microtechno9000** [comment]
> UPDATE: This is caused by (apparently) incorrect disk mounting, as on the few times it does not happen, the log shows the disk mounted successfully.
> 
> ```
> [05-13-2023 01:10:26] INFO ARM: Mounting disc to: /mnt/dev/sr1
> [05-13-2023 01:10:26] ERROR ARM: Mounting failed! Rip might have problems
> ```

Not sure why ARM raised issue with mounting, although I have seen it in my logs. But that isn't related to the UI bug. Resolved the issue in PR #848 with an update to the UI. It now just reports unknown as shown in the PR. 

Once the PR is completed this issue can be closed
